network/book.go

```diff
@@ -213,20 +227,20 @@ func (s *SimpleBook) slice(active bool) []*entry {
 	return entries
 }
 
-// byPriority type.
+// byPriority helps us sort peer entries by priority.
 type byPriority []*entry
 
-// Len method.
+// Len returns the count of peer entries..
 func (b byPriority) Len() int {
 	return len(b)
 }
 
-// Less method.
+// Less checks whether the score of the first peer is lower than the score of the second peer.
 func (b byPriority) Less(i int, j int) bool {
 	return b[i].score() < b[j].score()
 }
 
-// Swap method.
+// Swap will switch two peer entry positions in the list.
```

Comment:
Yes, but the linter will complain if there is no comment, and if you have to put one, might as well make it a sentence.